.idea/misc.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Raku v2020.02" project-jdk-type="Raku SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Raku v2019.07.1" project-jdk-type="Raku SDK" />
```

Comment:
Why the going back in version here?

Reply:
Thank you. That is my version of Comma, I think. The most uptodate one fails.
I will remove .idea files from being updated by git.

Reply:
I added .idea to .gitignore, but this seems to delete .idea in the main repository too.
Not sure how to eliminate this inconsequential part of the commit from the main changes below.
I could close the PR and open a new one?

Reply:
That question is above my git pay grade, I'm afraid :-)  So opening a new one seems safest.

Reply:
This is comma specific. The file itself should probably be eliminated.